src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,9 +1,10 @@
 package edu.kis.vh.nursery;
 
 import edu.kis.vh.nursery.structures.IntArrayStack;
+import edu.kis.vh.nursery.structures.IntStructureInterface;
 
 public class DefaultCountingOutRhymer {
-	private IntArrayStack arrStack;
+	private IntStructureInterface arrStack;
 
 	public DefaultCountingOutRhymer(IntArrayStack arrStack) {
```

Comment:
The constructor should take IntStructureInterface. 
The name of variables shouldn't indicate the data structure type arrStack -> stack